api/main_service/MainService/Controllers/PostsController.cs

```diff
@@ -162,15 +162,42 @@ public async Task<ActionResult<ResponseDto>> AddPost([FromForm] PostCreateDto ne
     [HttpPost("filter")]
     public async Task<ActionResult<PaginationResDto<IEnumerable<PostReadDto>>>> ViewPost(PaginationReqDto<PostFilterDto> pagination)
     {
+        var userId = "";
+
         // Create Post Filter
         var postFilter = Builders<Post>.Filter.Empty;
 
+        // Public Post Filter
+        postFilter = postFilter & Builders<Post>.Filter.Eq(x => x.DisplayStatus, PostStatus.PUBLIC);
+
+        // Published At Filter
+        postFilter = postFilter & Builders<Post>.Filter.Lte(x => x.PublishedAt, DateTime.Now);
+
+        // Get User Id
+        try
+        {
+            userId = User.FindFirst(JwtTokenPayload.USER_ID)!.Value;
+        }
+#pragma warning disable CS0168
+        catch (Exception e)
+#pragma warning restore CS0168
```

Comment:
What is this ?

Reply:
A trick to disable warning of catch Exception variable which doesn't use

Reply:
Did you remember that in C# try catch can be done without (Exception e) ?